ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -0,0 +1,91 @@
+!
+aaa group server ldap LADP2
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+aaa group server ldap LDAP1
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+radius-server attribute 32 include-in-access-req hostname
+radius-server dynamic-authorization port 1700
+radius-server tls ssl-profile GLOBAL_RADIUS_SSL_PROFILE
+radius-server dynamic-authorization tls ssl-profile SSL_PROFILE
+radius-server host 10.10.10.157 vrf mgt key 7 071B245F5A
+radius-server host 10.10.10.249 key 7 071B245F5A
+radius-server host 10.10.10.158 key 7 071B245F5A
+radius-server host 10.10.11.157 vrf mgt timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.159 vrf mgt retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.160 vrf mgt timeout 1 key 7 071B245F5A
+radius-server host 10.10.11.248 vrf mgt key 7 071B245F5A
+radius-server host 10.10.11.249 timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.158 timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.156 tls port 1700 timeout 1 retransmit 1
+radius-server host 10.10.11.155 vrf mgt tls ssl-profile HOST_SSL_PROFILE port 2083 timeout 1 retransmit 1
+!
+aaa group server radius RADIUS1
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+aaa group server radius RADIUS2
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+tacacs-server timeout 10
+tacacs-server host 10.10.10.157 single-connection vrf mgt key 7 071B245F5A
+tacacs-server host 10.10.10.249 timeout 23 key 7 071B245F5A
+tacacs-server host 10.10.10.158 key 7 071B245F5A
+tacacs-server host 10.10.10.159 key 8a $kUVyoj7FVQ//yw9D2lbqjA==$kxxohBiofI46IX3pw18KYQ==$DOOM0l9uU4TrQt2kyA7XCKtjUA==
+!
+aaa group server tacacs+ TACACS
+   server 10.10.11.157 vrf mgt
+   server 10.10.11.249
+!
+aaa group server tacacs+ TACACS1
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+aaa group server tacacs+ TACACS2
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+aaa authentication login default group TACACS local
+aaa authentication login console local
+aaa authentication enable default group TACACS local
+aaa authentication dot1x default group TACACS
+aaa authentication policy on-failure log
+aaa authentication policy on-success log
+aaa authentication policy local allow-nopassword-remote-login
+aaa authentication policy lockout failure 3 window 900 duration 300
+aaa authorization policy local default-role network-admin
+aaa authorization serial-console
+aaa authorization dynamic dot1x additional-groups group radius group RADIUS1
+aaa authorization exec default group TACACS local
+aaa authorization commands all default group TACACS
+aaa authorization commands 5 default group radius
```

Comment:
Getting this while configure the command on lab
% Authorization denied for command 'aaa authorization commands 5 default group radius': connect: Network is unreachable

Reply:
The line above enabled command authorization towards the TACACS server you have configured.

Reply:
yeap, you would either need to specify fallback method (like `local`) or test all this inside config session

Reply:
@alexeygorbunov so is the plan to test this on config session in the lab? or will it cause issues when trying to apply the config?

Reply:
We have talked about testing with cloudvision which uses a session.

Reply:
Yes, the plan is to run these tests via CloudVision, so this will not be an issue.